### DISCUSSION TITLE
How to use unified().use in JSON based .remarkrc

### DISCUSSION THREAD

**Discussion by gregoranders** [remarkjs/remark#949] [Q&A] [answered]
As a user I would like to use a JSON based .remarkrc file due to integration with several other tools. How do I use the following ESM based configuration in a JSON?

```js
...
[
  remarkRetext,
  unified().use({
    plugins: [
      retextEnglish
    ]
  })
]
...
```

**Comment by wooorm** [comment, marked as answer]
No, JSON cannot include JavaScript. You can change your file to a JavaScript file to import things and include JavaScript.

Not sure if it will ever land, but see also https://github.com/remarkjs/remark-retext/issues/14.
